src/extension/panel.ts

```diff
@@ -213,7 +213,7 @@ export class Panel {
 
     public selectByIndex(uri: Uri, runIndex: number, resultIndex: number) {
         const log = this.store.logs.find(log => log._uri === uri.toString());
-        const result = log?.runs[runIndex]?.results?[resultIndex];
+        const result = log?.runs?.[runIndex]?.results?.[resultIndex];
```

Comment:
>?. [](http://example.com/codeflow?start=32&length=2)

I just learn this in Typescript
that we will also need the "." if it is nullable.